src/sai/DivisibilityWhile.java

```diff
@@ -0,0 +1,73 @@
+package sai;
+
+/*
+ * Using while loop performing the divisibility by numbers
+ */
+
+class DivisibilityWhile{
+
+	void evenNumbers(int total){
+		System.out.println("Even numbers are :");
+		int count = 1;
```

Comment:
this extra variable not require, you need to create methods which accepts the two arguments.

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files